### DISCUSSION TITLE
how to select text in preview with mouse

### DISCUSSION THREAD

**Discussion by AnirudhG07** [sxyazi/yazi#1262] [1. Q&A] [answered]
## Configs
iTerm2, Shell - zsh, MacOS

## Problem
I donno how but I was able to drag and select text in yazi 0.2.5(downloaded from homebrew). But now I built from source for newer features, I coudnt do it. I tried all the 5 mouse_events, but nothing worked.

Can anyone tell how to select text in preview with mouse?

**Comment by sxyazi** [comment]
See https://github.com/sxyazi/yazi/issues/1132

**Reply by AnirudhG07** [reply]
Setting `mouse_events = [ ]` works. But I also want other mouse functionalities too. Pressing Shift and dragging doesnt work out for me.

**Reply by sxyazi** [reply, marked as answer]
> But I also want other mouse functionalities too.

You can't have them both at the same time; it's a limitation of the terminal itself.

> Pressing Shift and dragging doesnt work out for me

Most terminals support `Shift`, while some do not, up to the terminal implementation

**Reply by AnirudhG07** [reply]
Sad. It's fine I guess. Do you know which terminals do support it, just out of curiousity?
